web/src/billing/helpers.ts

```diff
@@ -163,7 +163,7 @@ export function set_tab(page: string): void {
 
 export function set_sponsorship_form(): void {
     $("#sponsorship-button").on("click", (e) => {
-        if (!is_valid_input($("#sponsorship-form"))) {
+        if (!is_valid_input($(".sponsorship-form"))) {
```

Comment:
This selector change doesn't seem to occur in `templates/corporate/sponsorship.html`

Reply:
Fixed in the commit.